model-optimizer/extensions/back/compress_quantized_weights.py

```diff
@@ -104,6 +104,12 @@ class CompressQuantizeWeights(BackReplacementPattern):
 
     force_clean_up = True
 
+    QUANTIZATION_MAP = {
+        # max_levels: (np_dtype, quantization_mode)
+        256: (np.int8, "signed"),
+        16: (packed_I4, "signed"),
```

Comment:
Is it possible that we need to have two ways to quantize with the same level? For example, 256 levels and both signed and unsigned mode. May be we should develop a more appropriate structure for this?

Reply:
No, there is no such use case. It is basically a switcher if we would consider packing int4 different way than int8.